main_routers/pages_router.py

```
@@ -23,9 +23,17 @@ async def get_default_index(request: Request):
 
 @router.get("/l2d", response_class=HTMLResponse)
 async def get_l2d_manager(request: Request):
-    """渲染Live2D模型管理器页面"""
+    """渲染模型管理器页面（兼容旧路由）"""
```

Ruff (0.14.11)

Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)